### DISCUSSION TITLE
How to detect Ribbon Tab click or Tab changed event?

### DISCUSSION THREAD

**Discussion by rulai-yinqiang** [harborsiem/WindowsRibbon#20] [Q&A] [answered]
My program has different controls in different tabs. I'd like to use Ribbon like TabControl.

How to detect Tab click or Tab changed event?

**Comment by harborsiem** [comment, marked as answer]
In Ribbon Framework there is no direct Tab click or Tab changed event.
But if you really need it, then you have to look to the EventLogger class which is placed inside the ribbon control. See Wiki for EventLogger.
